modules/multi-arch-uninstalling-using-cli.adoc

```diff
@@ -0,0 +1,75 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-uninstalling-using-cli_{context}"]
+
+= Uninstalling the Multiarch Tuning Operator by using the CLI
+
+You can uninstall the Multiarch Tuning Operator by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` permissions.
+* You have installed the {oc-first}.
+* You have deleted the `ClusterPodPlacementConfig` custom resource (CR).
```

Comment:
```suggestion
* You deleted the `ClusterPodPlacementConfig` custom resource (CR).
```
What happens if we don't delete the CR first? If we break something, it might still be worth an [IMPORTANT] admonition with the consequence.